en/docs/setup/changing-to-mariadb.md

```diff
@@ -1 +1,3 @@
-By default, WSO2 Identity Server uses the embedded H2 database as the database for storing user management and registry data. Follow the same steps and use the same database scripts, drivers, and configurations given in the documentation: [Changing to MySQL](../../setup/changing-to-mysql) in order to use MariaDB instead of the embedded H2 database.
+# Changing to Maria DB
+
+By default, WSO2 Identity Server uses the embedded H2 database as the database for storing user management and registry data. Follow the same steps and use the same database scripts, drivers, and configurations given in [Changing to MySQL](../../setup/changing-to-mysql) in order to use MariaDB instead of the embedded H2 database.
```

Comment:
```suggestion
By default, WSO2 Identity Server uses the embedded H2 database as the database for storing user management and registry data. Follow the same steps and use the same database scripts, drivers, and configurations given in [Changing to MySQL](../../setup/changing-to-mysql) to use MariaDB instead of the embedded H2 database.
```